test/json-shema-content-provider.test.ts

```diff
@@ -0,0 +1,44 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Red Hat, Inc. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as vscode from 'vscode';
+import { getDocUri, activate } from './helper';
+import * as assert from 'assert';
+
+describe('Tests for JSON Schema content provider', () => {
+  const SCHEMA = 'myschema';
+  const schemaJSON = JSON.stringify({
+    type: 'object',
+    properties: {
+      version: {
+        type: 'string',
+        description: 'A stringy string string',
+        enum: ['test'],
+      },
+    },
+  });
+
+  function onRequestSchema1URI(resource: string): string | undefined {
+    if (resource.endsWith('completion.yaml') || resource.endsWith('basic.yaml')) {
+      return `${SCHEMA}://schema/porter`;
+    }
+    return undefined;
+  }
+
+  function onRequestSchema1Content(): string | undefined {
+    return schemaJSON;
+  }
+
+  it('should handle "json-schema" url', async () => {
+    const docUri = getDocUri('completion/completion.yaml');
```

Comment:
If you run this test against windows you might need `${path.sep}`

Reply:
No, test on windows passed successfully

Reply:
ok